src/main/kotlin/de/itemis/mps/gradle/RunAntScript.kt

```diff
@@ -41,6 +42,10 @@ open class RunAntScript : DefaultTask() {
             }
         }
 
+        if(logging.level != LogLevel.LIFECYCLE && !allArgs.any { it.contains("mps\\.ant\\.log") }) {
+            allArgs = allArgs + "-Dmps.ant.log=${logging.level.toString().toLowerCase()}"
```

Comment:
Need to use `Locale.ENGLISH` with `toLowerCase()`, otherwise on a Turkish locale `I` in `INFO` would get lowercased to `ı` (dotless i).